production/helm/loki/values.yaml

```diff
@@ -203,6 +199,10 @@ initContainers: []
 #   image: busybox:latest
 #   command: ['sh', '-c', 'echo hello']
 
+## Additional Loki container arguments, e.g. log level (debug, info, warn, error)
+extraArgs: {}
```

Comment:
These were originally sorted alphabetically, though people have since diverged from that. Not sure why this churn is necessary.